__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const testData = require("../db/data/test-data")
+const seed = require("../db/seeds/seed")
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET/api/topics", () => {
+  test("200: return topics", () => {
+      return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual([
```

Comment:
You should use .objectContaining() to be able to check any amount of objects and their properties.